FirebaseSessions/Tests/Unit/InitiatorTests.swift

```diff
@@ -24,4 +24,26 @@ class InitiatorTests: XCTestCase {
     }
     assert(initiateCalled)
   }
+  
+  func testForegounding_initiatesNewSession() throws {
+    let pausedClock = ShadowDate()
+    let initiator = SessionInitiator(now: pausedClock.getDate)
+    var sessionCount = 0
+    initiator.beginListening {
+      sessionCount += 1
+    }
+    assert(sessionCount == 1)
+    
+    // Simulate 30 minutes + 1 second of backgrounding
+    initiator.appBackgrounded()
+    pausedClock.advance(by: 60 * 30 + 1)
+    initiator.appForegrounded()
+    assert(sessionCount == 2)
+    
+    // Simulate only 30 minutes of backgrounding
+    initiator.appBackgrounded()
+    pausedClock.advance(by: 60 * 30)
+    initiator.appForegrounded()
+    assert(sessionCount == 2)
```

Comment:
Could we add a comment explaining that the sessioncount should not increase because we incremented the time by less than the session background timeout?

Reply:
Done